test/unit/logging/Logger.spec.js

```diff
@@ -1,56 +1,51 @@
-const _last = require('lodash.last');
-
 describe('Logger', () => {
+  let logSpy;
 
   beforeEach(() => {
-    this.MockLogger = {
-      log: sinon.stub()
-    };
-
     injector()
-      .addDependency('console', this.MockLogger, true)
       .inject((Logger, console) => {
         this.Logger = Logger;
-        this.console = console;
 
-        this.getResult = () => {
-          return _last(this.console.log.getCalls()).args;
-        };
+        logSpy = jest.spyOn(console, 'log');
       });
   });
 
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
   it('should fatal()', () => {
     this.Logger.fatal('testing fatal');
-    expect(this.getResult()).to.deep.equal(['\u001b[31mLogger#fatal: \u001b[39m', 'testing fatal']);
+    expect(logSpy.mock.lastCall).toEqual(['\u001b[31mLogger#fatal: \u001b[39m', 'testing fatal']);
```

Comment:
This is super fragile, I would prefer to clear mock after every test and try `logSpy.toHaveBeenCalledWith`

Reply:
That's why we have 

```
  afterEach(() => {
    jest.resetAllMocks();
  });
```